tests/Participation/LotteryFilterTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace PhpAb\Participation;
+
+class LotteryFilterTest extends \PHPUnit_Framework_TestCase
+{
+    public function testShouldParticipateWithFullPropability()
+    {
+        // Arrange
+        $lottery = new LotteryFilter(100);
+
+        // Act
+        $participates = $lottery->shouldParticipate();
+
+        // Assert
+        $this->assertTrue($participates);
+    }
+
+    public function testShouldParticipateWithZeroPropability()
+    {
+        // Arrange
+        $lottery = new LotteryFilter(0);
+
+        // Act
+        $participates = $lottery->shouldParticipate();
+
+        // Assert
+        $this->assertFalse($participates);
+    }
+
+    public function testShouldParticipateWithCustomPropabilityAndPositiveResult()
```

Comment:
I'd like to check for mt_rand with non zero return. How can i add this since i can overwrite it only once in the namespace. Any ideas @waltertamboer ?